flutter-intellij-community.iml

```diff
@@ -44,5 +44,6 @@
     <orderEntry type="library" name="snakeyaml" level="project" />
     <orderEntry type="library" name="Dart SDK" level="project" />
     <orderEntry type="library" name="Dart Packages" level="project" />
+    <orderEntry type="library" name="jsr305" level="project" />
```

Comment:
I don't see these annotations used below?

Reply:
If not, consider reverting this dep (and the one below).

Reply:
Done.